config/jobs/kubernetes/sig-node/sig-node-config.yaml

```diff
@@ -21,7 +21,7 @@ periodics:
       - --gcp-zone=us-west1-b
       - --ginkgo-parallel=30
       - --provider=gce
-      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[NodeFeature:RuntimeHandler\] --minStartupPods=8
+      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Driver:.gcepd\]|\[Flaky\]|\[Feature:.+\]|\[NodeFeature:RuntimeHandler\] --minStartupPods=8
```

Comment:
It could be worth adding a comment here explaining why gcepd tests are disabled, or a TODO with an issue link.

Reply:
there are multiple places where we skip gcepd tests, I think it might be better to just track it where we need it to be resolved? https://github.com/kubernetes/cloud-provider-gcp/issues/293